simulator/cmd/simulator/simulator.go

```diff
@@ -40,13 +44,26 @@ func startSimulator() error {
 		Host: cfg.KubeAPIServerURL,
 	}
 	client := clientset.NewForConfigOrDie(restCfg)
+	dynamicClient, err := dynamic.NewForConfig(restCfg)
+	if err != nil {
+		return xerrors.Errorf("creates dynamic clientset: %w", err)
+	}
```

Comment:
nit: panicking may be sufficient

```suggestion
	dynamicClient := dynamic.NewForConfigOrDie(restCfg)
```